packages/components/tests/vue/components/word-usage-examples/word-usage-examples.test.js

```diff
@@ -27,7 +27,10 @@ describe('word-usage-examples.test.js', () => {
     testWordUsageList = await BaseTestHelp.collectConcordance(homonym)
   })
 
-  beforeEach(() => {
+  beforeEach/**
```

Comment:
maybe an ide error here with the inserted comments?

Reply:
Yes, I think it's a result of a glitch. Removed it.